cheatsheets/Third_Party_Javascript_Management_Cheat_Sheet.md

```diff
@@ -1,5 +1,18 @@
 # Introduction
 
+The attack surface of a web application is generally considered to be the size of the web site, aka the number of page urls, forms, including the number of parameters and parameter values for single page apps, as well as the amount of HTML generated by the server. Various definitions have also included the protocol between the client and the server (TLS). These are basically server side measures. A different measure includes the business logic complexity and the number of use cases or user stories on the site. A use case will involve a sequence of pages or URL parameter values, as well as the metadata of the user to initialize the use case; referrer, user agent, authorization or loyalty member level, language, geolocation etc.
+
+I define the use case attack surface as the combination of three values:
+UCAS1:the number of bytes of code executed for a given use case, user action or user story; (more code provides more risk of insecure code).
```

Comment:
Can you use bullet point for the different **UCAS** in order to made it more easy to read ?

Reply:
I agree, it felt strange to say 'I define',  I'll change it  and add bullets

Reply:
Also, I have never used Git, so I'll have to do some reading to figure out how to change existing text.

Reply:
I made the changes on the patch1 branch, I think.  Do I open a new pull request or do you approve the edited branch?

Reply:
You have the right thing, pushing to your fork update this PR so it's OK. 👍